scripts/generate-docs-admin.ts

```diff
@@ -23,7 +23,8 @@ Each component has general guidelines for usage as well as additional informatio
   `,
   },
   {
-    compileExamples: true,
+    compileExamples: false,
+    visibility: 'visible',
```

Comment:
we're not ready to use `compileExamples` in shopify-dev yet; got a bit ahead of myself here